src/Context/ApiContext.php

```diff
@@ -169,8 +169,8 @@ public static function restore($fileName = self::FILENAME_CONFIG_DEFAULT)
     /**
      * @param $fileName
      *
-     * @return string[][]
      * @throws BunqException
+     * @return string[][]
```

Comment:
the PSR-5 standard is still in debate and not accepted. I would suggest following @dnl-blkv suggestion